src/go/build/deps_test.go

```diff
@@ -382,6 +382,7 @@ var pkgDeps = map[string][]string{
 		"golang.org/x/net/lex/httplex",
 		"internal/nettrace",
 		"net/http/httptrace",
+		"regexp",
```

Comment:
There is not enough consensus on server push and how to do it, for this code to be even considered for upstream.
For now people can use it from our repo.